compact/far_backward/lumi/exit_window.xml

```diff
@@ -11,11 +11,19 @@
       name="LumiWindow"
       type="LumiWindow"
       material="Aluminum"
-      vis="AnlLightGray">
+      vis="AnlLightGray"
+      readout="LumiWindowCALHits">
       <dimensions x="LumiWin_R" y="LumiWin_R" z="LumiWin_thickness/2" />
       <position   x="0" y="0" z="LumiWin_Z" />
       <rotation   x="0*rad" y="0*rad" z="0*rad" />
     </detector>
   </detectors>
 
+  <readouts>
+    <readout name="LumiWindowCALHits">
+      <segmentation type="NoSegmentation"/>
+      <id>system:8</id>
+    </readout>
+  </readouts>
+
```

Comment:
Not sure if this is supposed to go into the official geometry. Why do we need this?

Reply:
Hi @veprbl ,
Thanks for your comment. The motivation for making the luminosity monitor exit window sensitive to radiation is that we have to study this element (a simple aluminum block) and check if it will survive under a high flux of synchrotron radiation photons. So far, the preliminary study showed that at 18 GeV, 0.227 A electron beam, the window will melt in 2 hours. So, we have to think about additional cooling. So, I would suggest keeping this part of the code to collect deposited energy in that element. 
What do you think?